packages/ui/src/primitives/field.tsx

```diff
@@ -65,7 +65,7 @@ export const LabelEnd = React.forwardRef(function Label(
   );
 });
 
-export const Optional = React.forwardRef(function Optional(
+export const Hint = React.forwardRef(function Hint(
```

Comment:
@joe-bell renamed to `Hint` here to make it useable outside of just using Optional in it.

Reply:
Makes sense to me, good shout